application/src/test/java/org/opentripplanner/apis/gtfs/mapping/routerequest/_RouteRequestTestContext.java

```diff
@@ -0,0 +1,123 @@
+package org.opentripplanner.apis.gtfs.mapping.routerequest;
+
+import static graphql.execution.ExecutionContextBuilder.newExecutionContextBuilder;
+import static java.util.Map.entry;
+import static org.opentripplanner.apis.gtfs.SchemaObjectMappersForTests.mapCoordinate;
+
+import graphql.ExecutionInput;
+import graphql.execution.ExecutionId;
+import graphql.schema.DataFetchingEnvironment;
+import graphql.schema.DataFetchingEnvironmentImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import org.locationtech.jts.geom.Coordinate;
+import org.opentripplanner._support.time.ZoneIds;
+import org.opentripplanner.apis.gtfs.GraphQLRequestContext;
+import org.opentripplanner.apis.gtfs.SchemaFactory;
+import org.opentripplanner.apis.gtfs.TestRoutingService;
+import org.opentripplanner.ext.fares.impl.DefaultFareService;
+import org.opentripplanner.routing.api.request.RouteRequest;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.routing.graphfinder.GraphFinder;
+import org.opentripplanner.service.realtimevehicles.internal.DefaultRealtimeVehicleService;
+import org.opentripplanner.service.vehicleparking.internal.DefaultVehicleParkingRepository;
+import org.opentripplanner.service.vehicleparking.internal.DefaultVehicleParkingService;
+import org.opentripplanner.service.vehiclerental.internal.DefaultVehicleRentalService;
+import org.opentripplanner.transit.service.DefaultTransitService;
+import org.opentripplanner.transit.service.TimetableRepository;
+
+class _RouteRequestTestContext {
```

Comment:
Why the underscore?

Reply:
It is used here to give a hint that this is a test framework class and not a test. It also sorts before all tests in the package. This follows the same notation as we already have for test framework packages . E.g. `o.o.raptor._data`, `o.o.raptor._support.arch` and `o.o._support`;